src/SigninState.ts

```diff
@@ -92,9 +93,9 @@ export class SigninState extends State {
         });
     }
 
-    public static fromStorageString(storageString: string): SigninState {
-        Logger.debug("SigninState.fromStorageString");
+    public static fromStorageString(storageString: string, clockService: ClockService): SigninState {
+        new Logger("SigninState").create("fromStorageString");
```

Comment:
You are right i now remember, i had it in a MR, but then dropped it. To avoid that in the future i will implement the `Logger.createStatic(staticMethod: string): Logger` logic. Will do so in a different MR.

With the one-liners of the static logging i have no problem, what triggered me was the cases were we have multiple logging within the same static function.

Reply:
see #425